src/safir/database.py

```diff
@@ -91,7 +91,19 @@ def datetime_from_db(time: None) -> None:
 
 
 def datetime_from_db(time: Optional[datetime]) -> Optional[datetime]:
-    """Add the UTC time zone to a naive datetime from the database."""
+    """Add the UTC time zone to a naive datetime from the database.
+
+    Parameters
+    ----------
+    time
+        The naive datetime from the database, or `None`
+
+    Returns
+    -------
+    datetime.datetime or None
```

Comment:
I noticed that the `datetime.datetime` gets linked, but the `None` doesn't. I think it's fine tbh, and explicitly marking up the `None` may not be worth it. I guess the rule is that the first "word" gets auto marked up, and the remaining words are treated as text.

Reply:
It's a bit more complex in my previous experiments: if you mark up `None`, then `datetime.datetime` doesn't get linked (unless you also mark it up). It's kind of weird.